java-source/src/main/java/net/corda/training/state/IOUState.java

```diff
@@ -48,38 +51,48 @@ public IOUState(Amount<Currency> amount, Party lender, Party borrower) {
     }
 
     public Party getLender() {
-        return lender;
+	    return lender;
```

Comment:
lots o' whitespace checks. There's an auto formatter in intelliJ which can do this for you

Reply:
Great tip - just ran the auto-formator on every Java file.